dod-advana/gamechanger-web
# Task/UOT-105062 - Organization Seal Images

## Description


- Add seal images to s3
- On the organization details page, admins can put in a manual override URL for the seal image
- New table created to support admin urls
- Add default DoD seal
- Seal images are on series of fallbacks: s3 -> admin override -> URL from graph -> default seal
- Fallback system applied to sidebar, organization details page, and card front
- Minor styling changes

## Related Issue/Ticket




https://jira.di2e.net/browse/UOT-109687
https://jira.di2e.net/browse/UOT-109688
https://jira.di2e.net/browse/UOT-109690
https://jira.di2e.net/browse/UOT-109736

## Testing instructions



New endpoints:
- /getOrgSeals
- /getOrgImageOverrideURLs
- /saveOrgImageOverrideURL

Set seal image URL as an admin by clicking edit icon in sidebar of Organization Details page
![image](https://user-images.githubusercontent.com/85301886/131566202-14fe72bc-0be0-44c2-bfa3-004c4f6c6b68.png)

Seal image should update on org details page, in sidebar, and on card
